examples/hcp-eks-demo/main.tf

```diff
@@ -54,46 +54,54 @@ module "eks" {
 }
 
 # The HVN created in HCP
-resource "hcp_hvn" "main" {
-  hvn_id         = var.hvn_id
-  cloud_provider = "aws"
-  region         = var.hvn_region
-  cidr_block     = var.hvn_cidr_block
+# resource "hcp_hvn" "main" {
+#   hvn_id         = var.hvn_id
+#   cloud_provider = "aws"
+#   region         = var.hvn_region
+#   cidr_block     = var.hvn_cidr_block
+# }
+
+data "hcp_hvn" "example" {
+  hvn_id = "hvn"
 }
```

Comment:
note-to-self: data source for HVN instead of creating HVN - temp for testing: revert this change